### DISCUSSION TITLE
Take existing output and convert?

### DISCUSSION THREAD

**Discussion by That-p4intp4rrot** [silverhack/monkey365#84] [Q&A]
So on a recent client I could not export the HTML report on the provided VDI. So my question is, is there a way to take the exported Excel or JSON reports and locally convert them using M365 into the much more detailed HTML report?

**Comment by silverhack** [comment]
Hi @That-p4intp4rrot, 

In the [latest](https://silverhack.github.io/monkey365/configuration/general-options/) version of monkey365 you can use the **Compress** option to compress all the output data into a single zip file. 

Hope that helps.
Cheers,
